pkg/templates/charts/toggle/cluster-backup/templates/managed-hub-placementrule.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  annotations:
+    "helm.sh/hook": pre-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+  name: acm-managed-hub-backup
+  namespace: open-cluster-management-backup
+spec:
+  clusterSelector:
```

Comment:
should we have here this as well ?

  clusterConditions:
    - status: 'True'
      type: ManagedClusterConditionAvailable

Reply:
This condition was removed in your previous PR and I don't have a managed hub to see if we really need it. I tested with a is-hub=true label only and the placement works. If you think we need it I can add it here too.

Reply:
this is to match what is now with the hub placement - try to place only on managed clusters which are in active state
I am not really sure what is the implication if we don't have this here ( if the managed cluster is not available, the hub can't connect to it anyway ). If we have this for the local-cluster , let's add it to the managed clusters too